File: NEsper/emit/EmitServiceProvider.cs
using System;
namespace net.esper.emit
{
	
	/// <summary> Static factory for implementations of the EmitService interface.</summary>
	public sealed class EmitServiceProvider
	{
		/// <summary> Creates an implementation of the EmitService interface.</summary>
		/// <returns> implementation
		/// </returns>
		public static EmitService newService()
		{
			return new EmitServiceImpl();
		}
	}
}

File: trunk/esper/src/main/java/com/espertech/esper/emit/EmitServiceProvider.java
package com.espertech.esper.emit;

/**
 * Static factory for implementations of the EmitService interface.
 */
public final class EmitServiceProvider
{
    /**
     * Creates an implementation of the EmitService interface.
     * @return implementation
     */
    public static EmitService newService()
    {
        return new EmitServiceImpl();
    }
}
